### ISSUE TITLE
Latex Build only works from root project file

### ISSUE THREAD

**Issue by maximelovino** [fannheyward/coc-texlab#96]
Hello,

I've switched to Vim recently and have discovered your coc extension for LaTeX. From what I've seen so far, it works great, thank you for your work.

I have a small gripe with it though, the `latex.Build` only seems to work correctly when working on the root file of a LaTeX project. I've setup the command using the default `latexmk` executable and the following configuration:
```json
{
	"latex.build.args": [
		"--shell-escape",
		"-pdf",
		"-interaction=nonstopmode",
		"-synctex=1"
	],
	"latex.build.onSave": true
}
```
My project structure is composed of a root file `report.tex`  and files for each chapters included in the main root document, the files are inside a folder, for example `chapters/chap1.tex`, `chapters/chap2.tex`.

I mainly work on those files, and when I save them or directly run the build command with `:CocCommand latex.Build` from these files it doesn't build anything. When I open the root file and run the command, everything builds correctly.

Thanks in advance for your help

**Comment by maximelovino** [comment]
Actually, it's okay, I found the problem. Actually only my main root file was recognized by vim as a `tex` file and thus starting the coc-texlab for that file. The other files were recognized as `plaintex` and coc-texlab was not available from them. I fixed this by adding:
```
let g:tex_flavor = "latex"
```
to my `.vimrc` file and now everything works from any file in the project. Perhaps we should add this to the README for new users?

**Comment by fannheyward** [comment]
I have add `plaintex` in LS support https://github.com/fannheyward/coc-texlab/blob/master/src/constants.ts#L18. 

You can `:CocUpdate` to upgrade `coc-texlab` to 1.9.7, and remove `let g:tex_flavor = "latex"`, to test if it works. If not, please give me feedback.

**Comment by maximelovino** [comment]
Now it seems to work ONLY on Plaintex files though and doesn't work on my root file or other files recognized as `tex`

**Comment by maximelovino** [comment]
If I activate the extension by hand from the extensions list, then everything works, so the problem is due to the automatic activation related to the filetypes in my opinion

**Comment by fannheyward** [comment]
Thanks for your feedback, I've published 1.9.8, can you give it a test?

**Comment by maximelovino** [comment]
Hello, thank you, yes it seems to work correctly now, activates on both tex and plaintex and  can build project from both type of files
